src/main/java/org/springframework/data/couchbase/core/ReactiveCouchbaseTemplateSupport.java

```diff
@@ -189,7 +189,7 @@ public void maybeEmitEvent(CouchbaseMappingEvent<?> event) {
 			try {
 				this.applicationContext.publishEvent(event);
 			} catch (Exception e) {
-				e.printStackTrace();
+				LOG.error("exception emitting event", e);
```

Comment:
you could add the event to the log message so in logging you can figure out which event caused the error